test/integration/puppetlabs/services/jruby/tasks_test.clj

```diff
@@ -89,6 +103,85 @@
                            {:name "init"
                             :module-name "apache"
                             :metadata? false
+                            :number-of-files 1}]
+              get-task-data (fn [env module task]
+                              (-> (.getTaskData jruby-puppet env module task)
+                                  mc/sort-nested-info-maps))]
+
+          (try (create-env env-1-dir env-1-tasks)
+            (testing "when the environment, module, and task do exist"
+              (testing "with the init task"
+                (let [res (get-task-data "env1" "apache" "init")]
+                  (is (nil? (schema/check mc/TaskData res)))))
+
+              (testing "with another named task"
+                (let [res (get-task-data "env1" "apache" "install")]
+                  (is (nil? (schema/check mc/TaskData res))))))
```

Comment:
These assertions should probably also make sure one piece of data is correct to make sure we got the right thing, like a file in the files list with "install" in the name